vhdbuilder/packer/list-images.sh

```diff
@@ -20,7 +22,8 @@ function generate_image_bom_for_containerd() {
     done
 
     IFS=$IFS_backup
-    jq --slurpfile images $TEMP_IMAGE_BOM_PATH -n '$images | group_by(.id) | map({id:.[0].id, repoTags:[.[].repoTags] | add | unique, repoDigests:[.[].repoDigests] | add | unique})' > $IMAGE_BOM_PATH
+    bom="$(jq --slurpfile images $TEMP_IMAGE_BOM_PATH -n '$images | group_by(.id) | map({id:.[0].id, repoTags:[.[].repoTags] | add | unique, repoDigests:[.[].repoDigests] | add | unique})')"
+    jq --argjson bom "$bom" --arg version "$IMAGE_VERSION" -n '{imageVersion:$version, imageBom:$bom}' > $IMAGE_BOM_PATH
```

Comment:
this quoting works right? I think so but sanity check

Reply:
works either way it looks like